### PR TITLE
OSDOCS-871 OSP supported versions for OCP 4.3

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.redhat.com/browse/OSDOCS-871

@racedo can you confirm this PR, #18962 (3.11), and #18963 (4.2) ? Thanks!

### CODE REVIEW

**Comment by codyhoag** [comment]
Approved by @racedo via Slack.

File: installing/installing_openstack/installing-openstack-installer-custom.adoc [hunk 1]
```diff
@@ -18,6 +18,21 @@ processes.
 
 * Have metadata service enabled in {rh-openstack}
 
+[IMPORTANT]
+====
+{product-title} 4.3 is supported for use with {rh-openstack} 13. {rh-openstack}
+16 will also be supported when it is released; it is currently in the RC phase
```
Comment (ahardin-rh) [inline installing/installing_openstack/installing-openstack-installer-custom.adoc]: What does RC stand for?

**Comment by ahardin-rh** [comment]
Just one question. Otherwise, looks good!

File: installing/installing_openstack/installing-openstack-installer-custom.adoc (hunk 1 above)
Comment (codyhoag) [reply installing/installing_openstack/installing-openstack-installer-custom.adoc]: @ahardin-rh it stands for *release candidate*. I can go ahead and add that with the abbreviation, for those that don't know what that means. E.g., `release candidate (RC)`.

**Pull request merged**